01-lets_get_started/01.rb

```diff
@@ -1,30 +1,34 @@
 # To make a empty square
 class Square
+  SYMBOL = '#'
   def initialize(side)
     @side = side
-    @gap_length = side - 2
   end
 
+  def generate
+    print_first_last_line
+
+    return if @side < 2
+    print_middle_lines
+    print_first_last_line
+  end
+
+  private
+
   def print_first_last_line
-    @side.times { print '#' }
+    print SYMBOL * @side
     print "\n"
   end
 
   def print_middle_lines
-    @gap_length.times { puts '#' + ' ' * @gap_length + '#' }
-  end
-
-  def generate
-    print_first_last_line
-
-    return if @side < 2
-        print_middle_lines
-        print_first_last_line
+    @gap_length = @side - 2
+    @gap_length.times { puts SYMBOL + ' ' * @gap_length + SYMBOL }
   end
 end
 
-print 'Enter the side length:'
+print 'side_length:'
```

Comment:
All printed messages should be standard English, thats simple & readable. 

Consider: `Side length: ` instead of `side_length`: